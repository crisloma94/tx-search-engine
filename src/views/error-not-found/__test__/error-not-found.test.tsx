import { render } from '@testing-library/react';
import Theme from '../../../styles/theme';
import { Provider } from 'react-redux';
import { store } from '../../../core/store/store';
import ErrorNotFound from '../error-not-found';

describe('Error not found view test suite', () => {
  const getComponent = () => (
    <Provider store={store}>
      <Theme>
        <ErrorNotFound />
      </Theme>
    </Provider>
  );
  const setup = () => render(getComponent());

  it('Should match snapshot', () => {
    const { container } = setup();
    expect(container).toMatchSnapshot();
  });
});
