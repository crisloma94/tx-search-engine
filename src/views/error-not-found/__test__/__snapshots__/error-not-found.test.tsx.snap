// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error not found view test suite Should match snapshot 1`] = `
<div>
  <div
    class="sc-bgqQcB kuwXLl"
  >
    <h1>
      Page not found
    </h1>
  </div>
</div>
`;
